packages/moonstone/styles/colors.less

```diff
@@ -230,4 +230,4 @@
 
 // Scrollable
 // ---------------------------------------
-@moon-scrollable-overscroll-color: @moon-accent;
+@moon-scrollable-overscroll-color: #ccc;
```

Comment:
Same as above. How about to use a variable?

Reply:
Same here as above. :) `#ccc` is used for other cases also, but it is not a universal color.